OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -31,8 +31,8 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Image to display.")]
 		public readonly string Image = null;
 
-		[Desc("Loop this sequence of Image while this projectile is moving.")]
-		[SequenceReference("Image")] public readonly string Sequence = "idle";
+		[Desc("Loop these sequences of Image while this projectile is moving.")]
+		[SequenceReference("Image")] public readonly string[] Sequences = { "idle" };
```

Comment:
Good to note, though. I was going to make it upgrade `Projectile: Shrapnel` to `Projectile: Bullet`, but if it hasn't reached playtest, the only users would likely be the official OpenRA mods.